save-backend/src/main/kotlin/com/saveourtool/save/backend/event/VulnerabilityMetadataListener.kt

```diff
@@ -0,0 +1,46 @@
+package com.saveourtool.save.backend.event
```

Comment:
please create events in vuln/event/ directory, so it will be easier to split it in the future

Reply:
Moved to a new directory